src/server/game.ts

```diff
@@ -66,6 +66,12 @@ export function removePlayer(socket_id: string) {
     return game.players;
 }
 
+// check if playername is unique
+export function isUniquePlayerName(player_Name: string){
+    player_Name= player_Name.toLowerCase();
+    return !game.players.find( player => player.playerName === player_Name);
```

Comment:
I found one bug while testing, as the player_name is converted to lowercase, it won't catch error if we put name again with lowercase, to avoid this, use
**player.playerName.toLowerCase()**